Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Dockerfile to build and serve scigateway-auth
+
+FROM python:3.6-alpine3.15
```

Comment:
Not building when using alpine :(

Reply:
@moonraker595 As already discussed earlier, the container was failing to build due to issues with the building and installation of the `cryptography` `pip` dependency. This was not the case when I opened this PR which is very confusing why it started doing this all of a sudden even on my machine. 

I made some changes in [b5f94cc](https://github.com/ral-facilities/scigateway-auth/pull/71/commits/b5f94cc22ccfae16284f568785ff66b1cbc93195) that fix this issue so could you please pull the latest changes on this branch and try to rebuild the container to see if it is happy on your machine?

Reply:
Builds fine and first time 👍